content/ko/docs/concepts/overview/working-with-objects/names.md

```diff
@@ -24,10 +24,20 @@ weight: 30
 
 {{< glossary_definition term_id="name" length="all" >}}
 
+**이름은 동일한 리소스의 모든 [API 버전](/docs/concepts/overview/kubernetes-api/#api-groups-and-versioning)에서 
+고유해야 한다. API 리소스는 API 그룹, 리소스 유형, 
+네임스페이스(네임스페이스 범위 리소스의 경우), 그리고 이름으로 구분된다. 즉, 이 맥락에서 API 버전은 고려 대상이 아니다.**
+
 {{< note >}}
 물리적 호스트를 나타내는 노드와 같이 오브젝트가 물리적 엔티티를 나타내는 경우, 노드를 삭제한 후 다시 생성하지 않은 채 동일한 이름으로 호스트를 다시 생성하면, 쿠버네티스는 새 호스트를 불일치로 이어질 수 있는 이전 호스트로 취급한다.
 {{< /note >}}
 
+리소스 생성 요청에 `generateName`이 `name` 대신에 제공되면 서버가 이름을 생성한다.
```

Comment:
```suggestion
리소스 생성 요청에서 `name` 대신 `generateName`이 제공될 경우 서버가 이름을 생성할 수 있다.
```

> The server may generate a name when `generateName` is provided instead of `name` in a resource create request.

원문의 느낌을 살려 자연스럽게 다듬어 보았습니다.